src/Core/src/Platform/Android/GlobalWindowInsetListener.cs

```diff
@@ -86,11 +86,14 @@ public GlobalWindowInsetListener() : base(DispatchModeStop)
             {
                 if (appBarLayoutContainsSomething)
                 {
-                    appBarLayout.SetPadding(leftInset, topInset, rightInset, 0);
+                    // Pad the AppBarLayout to avoid the navigation bar in landscape orientation and system UI in portrait.
+                    // In landscape, the navigation bar is on the left or right edge; in portrait, we account for the status bar and display cutouts.
+                    // Without this padding, the AppBarLayout would extend behind these system UI elements and be partially hidden or non-interactive.
+                    appBarLayout.SetPadding(systemBars?.Left ?? 0, topInset, systemBars?.Right ?? 0, 0);
```

Comment:
This changes how AppBarLayout handle horizontal insets.

Before: Applied `leftInset` and `rightInset` (which include navigation bar space)
With the changes:  Only applies `systemBars?.Left` and `systemBars?.Right` (excluding gesture navigation insets)

The code uses `systemBars?.Left/Right` but no use gesture navigation inset zones, the content is gestures zones works? For example, Pixel 9 in landscape with punch hole?

https://developer.android.com/reference/kotlin/androidx/compose/foundation/layout/package-summary#(androidx.compose.foundation.layout.WindowInsets.Companion).systemBars()